README.md

```diff
@@ -25,6 +25,7 @@ EOF
 make
 
 # Set the environment variables
+export OPERATOR_NAME=aws-efs-csi-driver-operator
```

Comment:
I already see `OPERATOR_NAME` on line 32

Reply:
I managed to miss that somehow, dropping this change.